example_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -13,13 +13,16 @@
 # DEFAULT_FROM_EMAIL = "..."
 
 ########################
-# DJANGO CMS SETTINGS
+# DJANGO_CMS
 ########################
 
-CMS_TEMPLATES = (
-    ('standard.html', 'Standard'),
-    ('fullwidth.html', 'Full Width'),
-)
+# CMS_TEMPLATES = (
+#     ('standard.html', 'Standard'),
+#     ('fullwidth.html', 'Full Width'),
+
+#     ('guide.html', 'Guide'),
+#     ('guides/portal_technology.html', 'Guide: Portal Technology Stack'),
+# }
```

Comment:
**Why?** In this example CMS, defaults are commented out to suggest you can change them. But defaults are not active, except for here. I am being consistent by commenting them out and showing the (current) defaults.